src/UseCases/Embed/EmbedUseCase.php

```diff
@@ -55,12 +56,28 @@ public function embed( string $fileUrl ): void {
 			return;
 		}
 
+		$renderer = $this->contentRendererFactory->createContentRenderer( $this->createRendererConfig( $request ) );
+
 		$this->presenter->showContent(
-			$this->contentRenderer->render(
+			$renderer->render(
 				$content,
-				$this->urlNormalizer->viewLevelNormalize( $fileUrl )
+				$this->urlNormalizer->viewLevelNormalize( $request->fileUrl )
 			)
 		);
+
+		$this->presenter->loadRendererRequirements( $renderer );
+	}
+
+	private function createRendererConfig( EmbedRequest $request ): RendererConfig {
+		return new RendererConfig(
+			fileExtension: $this->extractFileExtension( $this->urlNormalizer->viewLevelNormalize( $request->fileUrl ) ),
```

Comment:
should be fullNormalize, otherwise you won't find the extension of URLs like `https://github.com/ProfessionalWiki/ExternalContent`

Reply:
Both contain the extension when using the repo root URL. The full version just gives the raw URL.

fullNormalize: https://raw.githubusercontent.com/ProfessionalWiki/ExternalContent/master/README.md
viewLevelNormalize: https://github.com/ProfessionalWiki/ExternalContent/blob/master/README.md

Reply:
You sure about that? What happens when you use `https://github.com/ProfessionalWiki/ExternalContent` as input URL to the parser function? Looking at `GitHubUrlNormalizer`, I see defaulting for README.md in `normalizePath`, but not in `viewLevelNormalize`.

Reply:
I see you already changed it

Reply:
Yeah, with `{{#embed:https://github.com/ProfessionalWiki/ExternalContent}}` I get those two normalized URLs. But anyhow, I used `viewLevelNormalize` initially because that was what was passed into `ContentRenderer::render()`.